README.md

```diff
@@ -231,7 +231,6 @@ about the expectations for all contributors to this project.
 - Hans Kristian Flaatten ([starefossen](https://github.com/starefossen))
 - Hugues Malphettes ([hmalphettes](https://github.com/hmalphettes))
 - John Mitchell ([jlmitch5](https://github.com/jlmitch5))
-- Peter Petrov ([pesho](https://github.com/pesho))
```

Comment:
Could move this down to the "Emeritus" section if you want

Reply:
Done, thanks for the suggestion